### PR TITLE
[Showcase] Create users by PrimeHub SDK

### PR DESCRIPTION
**Pull request by LiuYuWei**
WHY:
1. Try to create lots of users by PrimeHub SDK.
HOW:
1. Add project into github repo.

### CODE REVIEW

File: sdk-create-users/src/batch_create_users.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    def __init_primehub(self, primehub_cluster_url, bool_primehub_config):
+        self.ph = PrimeHub(PrimeHubConfig())
+        if bool_primehub_config:
+            self.ph.config.generate(primehub_cluster_url)
+        elif self.ph.is_ready():
+            print("PrimeHub Python SDK setup successfully.")
+        else:
+            self.ph.config.generate(primehub_cluster_url)
+
```
Comment (jimmyliao) [inline sdk-create-users/src/batch_create_users.py]: [PEP8] [W293] blank line contains whitespace (line19)

**Comment by jimmyliao** [comment]
remember to cleanup .DS_Store from commit (sdk-create-users/data/.DS_Store)

**Comment by LiuYuWei** [comment]
@jimmyliao Please check the new commits. Thanks!

**Comment by jimmyliao** [comment]
> @jimmyliao Please check the new commits. Thanks!

@LiuYuWei from commit f9db793bd510c9ebb805b7f363fe0f6ff1145199 I didn't see your changes regarding the comment. I'd commit 916a60a0df05264d5ed2e7a3f27b0a0c207cb617 to make PEP compliant. Please take a look and see if you're ok with that. I will merge once get your confirmation. Thanks.

**Pull request merged**